helper/chart_data.js

```diff
@@ -181,49 +191,59 @@ async function instanceMultiselectFunc(data) {
 
 //Function for entity Observation and observation report's response creation
 exports.entityReportChart = async function (data,entityId,entityName,reportType) {
-    var obj;
-    var multiSelectArray = [];
-    var textArray = [];
-    var radioArray = [];
-    var sliderArray = [];
-    var numberArray = [];
-    var dateArray = [];
-    var noOfSubmissions = [];
-    var matrixArray = [];
+    let obj;
+    let multiSelectArray = [];
+    let textArray = [];
+    let radioArray = [];
+    let sliderArray = [];
+    let numberArray = [];
+    let dateArray = [];
+    let noOfSubmissions = [];
+    let matrixArray = [];
     let actualData = data;
+    let type;
```

Comment:
@deepakr123 - Please rename it to solutionType

Reply:
resolved